pkg/tenant/apikeys.go

```diff
@@ -6,6 +6,14 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
+func ProjectAPIKeyList(c *gin.Context) {
```

Comment:
Added `501 status` for now, so this doesn't need more than a glance. The same was implemented for the files remaining after this one.